service/src/test/kotlin/fi/espoo/evaka/invoicing/integration/EspooInvoiceIntegrationClientTest.kt

```diff
@@ -56,7 +58,7 @@ class EspooInvoiceIntegrationClientTest {
     @Test
     fun `sending an invoice uses a fallback address when the recipient's actual address is partially incomplete`() {
         val testInvoice = testInvoice(headOfFamily = testPerson(streetAddress = ""))
-        val batch = EspooInvoiceIntegrationClient.createBatchExports(listOf(testInvoice), testInvoice.agreementType, true)
+        val batch = EspooInvoiceIntegrationClient.createBatchExports(listOf(testInvoice), testInvoice.agreementType!!, true)
```

Comment:
Is there a reason why this is `testInvoice.agreementType!!` instead of `agreementType` like in other tests?

Reply:
:man_facepalming: it's just a refactoring mistake...I initially had `!!` everywhere but refactored it and missed this place